src/bin/solang.rs

```diff
@@ -202,9 +202,19 @@ fn main() {
                             .hide(true),
                     )
                     .arg(
-                        Arg::new("LOGRUNTIMEERRORS")
-                            .help("Log runtime errors in the environment")
-                            .long("log-runtime-errors")
+                        Arg::new("NOLOGRUNTIMEERRORS")
+                            .help("Disable logging runtime errors in the environment")
+                            .long("no-log-runtime-errors")
+                            .action(ArgAction::SetFalse),
+                    ).arg(
+                        Arg::new("NOPRINT")
+                            .help("Disable logging prints in the environment")
+                            .long("no-print")
+                            .action(ArgAction::SetFalse),
+                    ).arg(
+                        Arg::new("RELEASE")
+                            .help("Disable all debugging features such as prints, logging runtime errors, and logging api return codes")
+                            .long("release")
                             .action(ArgAction::SetTrue),
```

Comment:
Does `SetTrue` mean it will be disabled by default? 
I am not familiar with these commands.

Reply:
`SetTrue` means that the variable `release` will be set to true if the argument was passed to the command